glide.lock

```diff
@@ -25,18 +25,7 @@ testImports:
   subpackages:
   - assert
   - require
-- name: go.uber.org/atomic
-  version: 4e336646b2ef9fc6e47be8e21594178f98e5ebcf
 - name: go.uber.org/tools
   version: ce2550dad7144b81ae2f67dc5e55597643f6902b
   subpackages:
   - update-license
-- name: go.uber.org/zap
```

Comment:
this lock change definitely looks like a win. We don't want to pull in zap for people who want to use the config package